manifest.json

```diff
@@ -5,11 +5,13 @@
 	"description": "A tool that you have to setup just one time, to automatically login to your ERP every time.",
 	"version": "1.9",
 	"browser_action": {
-		"default_icon": "iitkgp.png",
+		"default_icon": "logo16.png",
 		"default_popup": "popup.html"
```

Comment:
@the-halfbloodprince Do you think it would be a good idea to move all the new images into a folder "assets"? This could be an opportunity to rename the "instruction_images" to "assets" and move all images into a single folder and update the references wherever they are used (`README.md` most probably).